picard/util/__init__.py

```diff
@@ -611,3 +611,12 @@ def find_best_match(candidates, no_match):
     else:
         result = no_match
     return BestMatch(similarity=result.similarity, result=result, num_results=len(sorted_results))
+
+
+def get_qt_enum(cls, enum):
```

Comment:
We should document this function, it's simple, but its usage isn't obvious. An example could do.

Reply:
Done